apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -1072,6 +1072,23 @@ function handleAppBannerShowing( calypsoPort ) {
 	};
 }
 
+function handleHelpCenterShowing( calypsoPort ) {
+	const { port1, port2 } = new MessageChannel();
+
+	calypsoPort.postMessage(
+		{
+			action: 'getIsHelpCenterShown',
+			payload: {},
+		},
+		[ port2 ]
+	);
+
+	port1.onmessage = ( { data } ) => {
+		const { isHelpCenterVisible } = data;
+		dispatch( 'automattic/help-center' ).setShowHelpCenter( isHelpCenterVisible );
```

Comment:
This can be undefined in case of a race condition. 
```suggestion
		dispatch( 'automattic/help-center' )?.setShowHelpCenter?.( isHelpCenterVisible );
```